UKHomeOffice/cop-ui
# fixed number of your team's tasks not matching

## Description
This PR fixes the number of tasks displayed in the COP Dashboard not previously matching the number of tasks displayed in your teams tasks.

## To Test
On the COP Dashboard ensure the following : 

- **Step 1**: There is one or more tasks assigned to your team
 
- **Step 2**: Locate where it says **tasks assigned to your team**, note the number shown above that piece of text and then click on the number.

- **Step 3**: In the new page that is loaded, locate the text **Your team's tasks**, confirm that the number below that matches the number in **Step 2**.



** PR on behalf of Ian Brandon
